tests/helpers.py

```diff
@@ -85,25 +85,22 @@ def wait_for_resource_state(resource, state, timeout=60):
         logger.error(f"{resource.kind} {resource.name} failed to reach {state}")
         resource.reload()
         logging.error(f"\n{resource.describe()}")
-        return False
+        raise
```

Comment:
1 samll change,
Can we raise the output from resource.describe()?
It will help a lot for people to see it directly in jenkins test results instead of going to console output

Reply:
yes, done